common/views/components/styled/SpacingComponent.tsx

```diff
@@ -18,7 +19,7 @@ const SpacingComponent = styled.div.attrs({
       `)}
   }
 
-  @supports selector(:has(a)) {
+  &.text + &.text {
```

Comment:
I'm just wondering if we should prefix these with e.g. `slice-type-` to prevent any potential clashes?